x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -113,6 +123,16 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
       ];
     }, [os]);
 
+    // use a memoized callback to
+    // limit list iteration for re-renders
+    const operatorOptions = useMemo<Array<EuiSuperSelectOption<string>>>(() => {
+      return (Object.keys(OperatorEntryField) as [ConditionEntry['operator']]).map((value) => ({
+        dropdownDisplay: OPERATOR_TITLES[value],
+        inputDisplay: OPERATOR_TITLES[value],
+        value,
+      }));
+    }, []);
```

Comment:
`operatorOptions` doesn't depend on the component props. It's better to move it outside the component and omit `useMemo` altogether.